docker/lite/Dockerfile

```diff
@@ -18,13 +18,20 @@
 
 # Use a temporary layer for the build stage.
 ARG bootstrap_version=31
-ARG image="vitess/bootstrap:${bootstrap_version}-mysql80"
+ARG image="vitess/bootstrap:${bootstrap_version}-common"
```

Comment:
This was reverted back to `common` as we no longer need to copy `mysqlbinlog` from the bootstrap image.